### DISCUSSION TITLE
Breakpoints - max-width - add Comparison Operators explanation

### DISCUSSION THREAD

**Discussion by asidelnik** [mdn/mdn-community#345] [Content]
**Idea - add Comparison Operators explanation to breakpoints docs:**
`max-width(selected length)` is similar to `if(screen length <= selected length)`

- Can be added to the "Beginner's guide to media queries".
- Could be useful for JS developers

**Comment by bsmth** [comment]
Thanks for reporting. There's something on that doc now about ranged values in this format:

```css
@media (30em <= width <= 50em) {
  /* … */
}
```

* https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/CSS_layout/Media_queries#using_ranged_syntax

I'm going to close as resolved, but if you think there's still room for improvement, let me know. We'd also accept a PR for improvements here:

https://github.com/mdn/content/blob/main/files/en-us/learn_web_development/core/css_layout/media_queries/index.md

Thanks!
